worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/XmlFieldEntry.java

```diff
@@ -18,7 +18,7 @@
 public class XmlFieldEntry
 {
     private final String name;
-    private final String text;
+    private String text;
```

Comment:
If you updated the String content of the field instead of modifying xml element you wouldn't need to change this.... I think there was a reason to make this class immutable, don't remember exactly what it was now...

Reply:
See previous comment.